src/spatial/math/mod.rs

```diff
@@ -100,3 +101,41 @@ pub(crate) fn set_occupancy_in_bitmap_64bits(
 pub(crate) fn octant_bitmask(octant: u8) -> u8 {
     0x01 << octant
 }
+
+#[cfg(feature = "dot_vox_support")]
+pub(crate) enum CoordinateSystemType {
+    LZUP, // Left handed Z Up
+    LYUP, // Left handed Y Up
+    RZUP, // Right handed Z Up
+    RYUP, // Right handed Y Up
+}
+
+#[cfg(feature = "dot_vox_support")]
+pub(crate) fn convert_coordinate<T: Copy + Neg<Output = T>>(
+    c: V3c<T>,
+    src_type: CoordinateSystemType,
+    dst_type: CoordinateSystemType,
+) -> V3c<T> {
+    match (src_type, dst_type) {
+        (CoordinateSystemType::LZUP, CoordinateSystemType::LZUP) => c,
+        (CoordinateSystemType::LYUP, CoordinateSystemType::LYUP) => c,
+        (CoordinateSystemType::RZUP, CoordinateSystemType::RZUP) => c,
+        (CoordinateSystemType::RYUP, CoordinateSystemType::RYUP) => c,
```

Comment:
If you derived `PartialEq` on `CoordinateSystemType` you could go with:

`(src, dst) if src == dst => c`

instead of the 4 patterns where the two arguments are the same.

Just saying, it's not important, just saw it.

Reply:
for some reason it gives non-exhaustive patterns.. I guess the parser doesn't recognize it as exhaustive this way.. ?! 
This is the only reason I am not introducing this, but the suggestion was good!